src/components/styledComponents.jsx

```diff
@@ -0,0 +1,35 @@
+import styled, { createGlobalStyle } from "styled-components";
+import reset from 'styled-reset';
+
+const GlobalStyle = createGlobalStyle`
+  ${reset};
+`;
+
+const HourCell = styled.div`
+  position: relative;
+  display: inline-block;
+  height: 10vh;
+  width: 15wh;
+  justify-content: start;
+`
```

Comment:
And then for each individual component, we can just write the css as normal here. Then we drop it into the code like any other react component.